src/modules/Verifications/components/TypeRadioGroup/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { Radio, RadioGroup } from '@mui/material';
+import React from 'react';
+import check from 'src/assets/icons/item-check.svg';
```

Comment:
Don't we have this icon in our icons?

Reply:
it is just used here, we can add it to icon pack later, I will add a FIXME tag

Reply:
we already have it 
you can use "tick" icon inside a square div element